students/nskvarch/Lesson6/mailroomtest.py

```diff
@@ -0,0 +1,126 @@
+#!usr/bin/env python3
+# Unit Test script for the mailroom exercise created by Niels Skvarch
+
+from mailroom4 import print_report, run_report, thanks, print_thankyou, create_thankyou_all, add_new_donor
+import os
+import unittest
+import io
+from contextlib import redirect_stdout
+from unittest import mock
+import sys
+
+test_datalines = [('Name', 'Total Donated', 'Times Donated', 'Average Donation'),
+                  ('Old Gregg', '16396.1', '3', '5465.366666666666'), ('Bob Johnson', '4284.49', '2', '2142.245'),
+                  ('Fred Billyjoe', '1356.28', '3', '452.0933333333333'),
+                  ('Jerry Vars', '772.6100000000001', '3', '257.5366666666667'), ('Harry Richard', '1.5', '1', '1.5')]
+
+
+class test_case_one(unittest.TestCase):
+    def test_print_report(self):
+        test_report = ("""Name             $    Total Donated        Times Donated    	     Average Donation
+Old Gregg        $    16396.1                          3    	    5465.366666666666
+Bob Johnson      $    4284.49                          2    	             2142.245
+Fred Billyjoe    $    1356.28                          3    	    452.0933333333333
+Jerry Vars       $    772.6100000000001                3    	    257.5366666666667
+Harry Richard    $    1.5                              1    	                  1.5
+
+
+""")
+        output = io.StringIO()
+        with redirect_stdout(output):
+            print_report(test_datalines)
+        output.seek(0)
+        output1 = output.read()
+        assert output1 == test_report
+
+
+class test_case_two(unittest.TestCase):
+    def test_run_report(self):
```

Comment:
your tests should all live in a single class, and have multiple functions for each test case:
```
class TestMailroom(unittest.TestCase):
    def test_report(self):
        pass

    def test_add_donor(self):
        pass
```